go.mod

```diff
@@ -24,3 +23,8 @@ require (
 	k8s.io/klog v1.0.0
 	sigs.k8s.io/controller-runtime v0.5.2
 )
+
+replace (
```

Comment:
This was required to handle issues with the azure auth provider.  Copied from the helm go.mod https://github.com/helm/helm/blob/8a4aeec08d67a7b84472007529e8097ec3742105/go.mod#L47-L50